### ISSUE TITLE
latest tag is 1.1.109-0 not 2.x

### ISSUE THREAD

**Issue by wdl1908** [factoriotools/factorio-docker#551]
when pulling latest tag from docker hub it give you the version 1.1.109-0 not 2.x

**Comment by MaxDev98** [comment]
Does work for me. ;)

Just updated and pulled the image and it returned version 2.0.17 for `docker image inspect factoriotools/factorio`

**Comment by wdl1908** [comment]
Strange when I added the latest tag explicitly it downloaded the 1.1 version

**Comment by Fank** [comment]
Did you pulled the latest tag before?
It does not auto pull the tag information, if you ever used latest tag before like when it was version 1.1.109 it reuses it.

**Comment by wdl1908** [comment]
Yes it seems I pulled that latest 2 months ago so closing as N/A
